xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1497,7 +1497,7 @@ void CGUIWindowVideoBase::LoadPlayList(const std::string& strPlayList, int iPlay
     // load it
     if (!pPlayList->Load(strPlayList))
     {
-      CGUIDialogOK::ShowAndGetInput(6, 0, 477, 0);
+      CGUIDialogOK::ShowAndGetInput(6, 477);
```

Comment:
I had seen that string in the strings.po file and always wondered where it's used. The usage in here is utterly wrong though. So let's either change that label (I suppose it's some XBOX leftover) or use a different string here (like 16026 -> "Playback failed").